iati/core/tests/test_default.py

```diff
@@ -108,6 +109,22 @@ def test_default_codelists_length(self, codelist_lengths_by_version):
         assert len(codelists) == codelist_lengths_by_version.expected_length
 
 
+class TestDefaultRulesets(object):
+    """A container for tests relating to default Rulesets."""
+
+    def test_default_ruleset(self):
+        """Check that the default Ruleset is correct.
+
+        Todo:
+            Handle multiple versions.
```

Comment:
Need to handle multiple versions - can use `standard_version_optional` to do this.

Reply:
done